kvmagent/kvmagent/plugins/ovn.py

```
@@ -188,46 +197,85 @@ def start_ovn_service(self, req):
+                interfaces_names = []
+                for interface in interfaces:
+                    interfaces_names.append(vsctl.getTableAtrr("Interface", interface, "name"))
+                if set(interfaces_names) == set(cmd.nicNamePciAddressMap.items().values()):
+                    for nicName, pciAddress in cmd.nicNamePciAddressMap.items():
+                        err, pci = vsctl.getTableAtrr("Interface", nicName, "options:dpdk-devargs")
+                        if err or pci != pciAddress:
+                            needAddUplink = True
+                            break
+                        err1, nicRxQueueNum = vsctl.getNicRxQueueNumConfig(nicName)
+                        err2, nicRxQueueDescNum = vsctl.getNicRxQueueDescNumConfig(nicName)
+                        if err1 or err2 or  nicRxQueueNum != cmd.nicRxQueueNum or nicRxQueueDescNum != cmd.nicRxQueueDescNum:
```

Flake8 (7.2.0)

[error] multiple spaces after keyword

(E271)